projects/valtimo/dossier/src/lib/components/dossier-detail/tab/widgets/components/collection/widget-collection.component.scss

```diff
@@ -14,6 +14,10 @@
  * limitations under the License.
  */
 
+label {
+  max-width: 80vh;
```

Comment:
Why vh? Could you use percent?

Reply:
I tried it before but if I use percent, the div is not resized

Reply:
This does not handle the overflow properly. @mbritense suggested to use this:
```
white-space: nowrap;
overflow: hidden;
text-overflow: ellipsis;
```